src/App.tsx

```diff
@@ -1,24 +1,52 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useContext } from 'react';
+import { TodoList } from './components/TodoLIst/TodoLIst';
+import { Footer } from './components/Footer/Footer';
+import {
+  ErrorNotification,
+} from './components/ErrorNotification/ErrorNotification';
+import { TodosContext } from './components/TodosContext/TodosContext';
+import { Header } from './components/Header/Header';
+import { FilterItem } from './types/FilterItem';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const { todos, filter } = useContext(TodosContext);
+
+  const activeTodos = todos?.filter(todo => !todo.completed);
+  const completedTodos = todos?.filter(todo => todo.completed);
+
+  function getFilterTodos() {
+    switch (filter) {
+      case FilterItem.Active: {
+        return activeTodos;
+      }
+
+      case FilterItem.Completed: {
+        return completedTodos;
+      }
+
+      default:
+        return todos;
+    }
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">React Todo App - Load Todos</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <Header />
+
+        <TodoList todos={getFilterTodos()} />
```

Comment:
Create variable and use here  `const  visibleTodos = getFilterTodos();`
```suggestion
        <TodoList todos={visibleTodos} />
```